shell/components/PromptModal.vue

```diff
@@ -38,6 +38,11 @@ export default {
     cssProps() {
       // this computed property lets us generate a scss var that we can use in the style
       return `--prompt-modal-width: ${ this.modalWidth }`;
+    },
+    stickyProps() {
+      const isSticky = !!this.modalData?.modalSticky;
+
+      return !isSticky ? '' : ';display: flex; flex-direction: column;';
```

Comment:
since border-radius already ends with `;` I don't think we need the first `;` on `;display: flex; flex-direction: column;`. A bit afraid that some exotic browser might not parse this correctly 🤔

Reply:
Fixed